wiki/Beatmap_Editor_Guides/Changing_Title_Text/en.md

```diff
@@ -1,33 +1,36 @@
+Changing Title Text
+=====================
+
 [*Changing the Title Text : The Guide* by: Ekaru](https://osu.ppy.sh/forum/t/14513)
 
 How to get it changed
-=====================
+-------------------------
 
 Sadly, you can't do this yourself. Just ask a QAT to change it. They're the only ones that are able to, so just give them a note of what you need/want, and if it's reasonable enough, you'll get your new title text on your map! Easy, right?!
 
 Why change?
-===========
+------------
 
 If used correctly, it can add a certain effect (in a good way!) to a map. It changes the presentation of your map in an effective manner if used correctly by changing the elements and the time of the appearance of the title. If you're a storyboarder, this might be a very huge benefit to you!
 
 -   An effective example would be [Shiirn's Solar map](http://osu.ppy.sh/s/33483)
 -   You can split the title if it's too long for a single line: [example](https://osu.ppy.sh/s/1391), [another one](https://osu.ppy.sh/s/57560)
 
 General info
-============
+-------------
 
 Hand this to your favourite QAT.
 
 -   Syntax is as follows:
 
-`[firstOption:firstValue,secondOption:secondValue] and so on, so forth; notice there are no spaces around punctuation marks`
+`[firstOption:firstValue,secondOption:secondValue]` and so on, so forth; notice there are no spaces around punctuation marks
```

Comment:
L26: Suggestion: add spacing ``[firstOption : firstValue, secondOption: secondValue]`` for readability